src/Controller/FirstController.php

```diff
@@ -2,19 +2,26 @@
 
 namespace App\Controller;
 
-use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 use Symfony\Component\HttpFoundation\JsonResponse;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\Routing\Annotation\Route;
 
-class FirstController extends AbstractController
+class FirstController
 {
     /**
-     *@Route("/message")
+     * @Route("/message", methods={"GET"})
      */
-    public function getMessage(Request $request): Response
+    public function getMessage(): Response
     {
         return new JsonResponse("One Eternety Later...", Response::HTTP_OK, []);
```

Comment:
We were recommended by Calin not to use double quotes, unless it's a more complex string. Otherwise looks great